src/universal-dapp.js

```diff
@@ -514,6 +524,7 @@ UniversalDApp.prototype.getCallButton = function (args) {
       } else {
         clearOutput($result);
         $result.append(getReturnOutput(result)).append(getGasUsedOutput(result));
+        $result.append(getDebugTransaction(result));
```

Comment:
This should be present in other cases, too - we want to debug constant functions.